### ISSUE TITLE
WPA-Enterprise (802.1x) support?

### ISSUE THREAD

**Issue by jackassplus** [bettercap/bettercap#555]
Feature request - WPA-Enterprise (802.1x) support

### Environment

Please provide:

* bettercap v2.23 (built for darwin amd64 with go1.11.1)
* MacOS 10.14
* http-ui caplet or ticker 

**Expected behavior:** *What you expected to happen*
Ideally, in addition to "WPA2" it would show authentication like airodump-ng. (bonus points for  which variant e.g. EAP-TLS, EAP-TTLS)

Also capture certificates and EAP frames in the .pcap

**Comment by evilsocket** [comment]
If you're using the UI, did you try to mouse hover that WPA2?

**Comment by jackassplus** [comment]
I did not. Didn't occur to me. I'll give it a shot as soon as I can get it to allow me to choose en0 as a wifi interface.

**Comment by evilsocket** [comment]
just disconnect it from any network and you'll be able to use it in monitor mode

**Comment by Touseef-ahmad** [comment]
bettercap doesn't work on 802.1x on kali linux

**Comment by l3g0l4s-exe** [comment]
Are WPA-Enterprise and WPA2-Enterprise still unsupported on 2.4.0? I didn't find any reference on the repository.

**Comment by evilsocket** [comment]
can you guys clarify what do you mean with support? The handshake captures? Simply the detection that a network is 802.1x? Because 2.4.0 should show the difference now.

**Comment by l3g0l4s-exe** [comment]
> can you guys clarify what do you mean with support? The handshake captures? Simply the detection that a network is 802.1x? Because 2.4.0 should show the difference now.

You're right, sorry for the limited information provided. I don't know the @jackassplus and @Touseef-ahmad use cases, I would like bettercap to support/integrate 802.1x (and so EAP and RADIUS) attacks like the ones already integrated for 802.11 networks. For example, it would be awesome to be able to manipulate certificates on the fly, to intercept MSCHAP hashes, and to spawn rogue APs for 802.1x EvilTwin. I know that these tasks can be accomplished with a bunch of tools, but I like to have a centralized, extensible, swiss-knife like bettercap.

**Comment by jackassplus** [comment]
This was forever ago. I have not looked at this in a while. I do, however,
concur with l3g0l4s.
